src/main/java/com/xebialabs/overthere/util/WindowsCommandLineArgsSanitizer.java

```diff
@@ -24,9 +24,18 @@
 
 public class WindowsCommandLineArgsSanitizer {
 
-    private static final char[] SPECIAL_CHARS = "|<>&^\r\n".toCharArray();
+    private static final char[] SPECIAL_CHARS = " |<>&^\r\n".toCharArray();
```

Comment:
Ok, now the 1 million dollar question. If a string contains spaces _and_ special characters, we're now not escaping any carets. Isn't it good enough then to _always_ put quotes around the string?